### DISCUSSION TITLE
How to sync with current Firestore ?

### DISCUSSION THREAD

**Discussion by ekimcem** [firecmsco/firecms#542] [Q&A]
Hello everyone,

I already have a firestore. I want to create a firecms app that will connect to my db directly and show my collections.

How can i fetch the current db to CMS ?

**Comment by fgatti675** [comment]
Hi @ekimcem 
FireCMS simply connects to your Firestore instance.
Just follow the quickstart to get started :)
https://firecms.co/docs/quickstart
